src/components/pages/Landing/Component.js

```diff
@@ -1,31 +1,85 @@
 import React from 'react';
 import { Canvas } from '../../layouts';
 import { SearchPanel } from '../../common/';
+import { PlotlyCard } from '../../common';
+import CardContainer from '../../common/CardContainer';
+import '../../../styles/LandingPage.less';
 
 // these functions are acting as jsx return statements, can be compartmentalized later.
 function titleContainer() {
   return <div></div>;
 }
 function title() {
   return (
-    <div
-      id="title"
-      style={{
-        fontSize: 36,
-        paddingTop: '30vh',
-        lineHeight: '48px',
-        fontStyle: 'normal',
-        fontWeight: 'normal',
-        color: '#fff',
-        textAlign: 'center',
-      }}
-    >
-      {' '}
-      Where will you wander?
+    <div className="landing-container">
+      <div className="title">
+        {' '}
+        Welcome To Citrics <br></br> City Comparison
+        <SearchPanel />
+      </div>
+
+      <h3>Popular Comparisons:</h3>
+      <section>
+        <div className="pop-item">
+          <div className="pop-pic">
+            <img
+              src="https://cdn.pixabay.com/photo/2016/11/22/19/17/architecture-1850129_1280.jpg"
+              alt="New York City"
+            />
+          </div>
+          <p>New York, NY</p>
+          {/* <CardContainer Card={PlotlyCard} /> */}
```

Comment:
Why is the PlotlyCard commented out? Are you planning on incorporating later on?

Reply:
Good catch, will delete that